uffi-compat/uffi-compat.lisp

```diff
@@ -628,6 +632,12 @@ output to *trace-output*.  Returns the shell's exit code."
           `((with-foreign-strings ,(cdr bindings) ,@body))
           body)))
 
+#+ecl
+(defun foreign-string-length (foreign-string)
+  (declare (inline ffi:foreign-string-length))
+  (ffi:foreign-string-length foreign-string))
```

Comment:
Ecl seems to use these when its compiled w/o dffi support. It crossed my mind to not add it there - if nobody lack this, then its ok with me - its accessible via native ffi anyway